src/nulog-train/nulog_train.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: nulog-train
+spec:
+  template:
+    metadata:
+      labels:
+        app: nulog-train
+    spec:
+      containers:
+      - name: nulog-train
+        image: amartyarancher/nulog-train
```

Comment:
add specific tag `v0.0` to `amartyarancher/nulog-train`... should be `amartyarancher/nulog-train:v0.0`